### PR TITLE
Added `price` & `currencyCode`

### PR DESCRIPTION
**Pull request by yzlin**
Add new fields `price` & `currencyCode` without digging into `skProduct` or `skuDetail`.
This resolves part of #26.

### CODE REVIEW

**Comment by suriksarkisyan** [comment]
Hi, @yzlin 
Awesome work, thank you so much.
I have few comments, after fixes, we will make a release ASAP.

File: ios/EntitiesConverter.m [hunk 1]
```diff
@@ -88,6 +88,7 @@ + (NSDictionary *)convertSKProduct:(SKProduct *)product {
     result[@"isDownloadable"] = @(product.isDownloadable);
     result[@"downloadContentVersion"] = product.downloadContentVersion;
     result[@"downloadContentLengths"] = product.downloadContentLengths;
+    result[@"currencyCode"] = product.priceLocale.currencyCode;
```
Comment (suriksarkisyan) [inline ios/EntitiesConverter.m:91]: Our SDK supports iOS from version 9.0. `currencyCode` available from iOS 10.
In this case, iOS 9 will crash. Could you please add the availability check as added below? 🙏

Comment (yzlin) [reply ios/EntitiesConverter.m:91]: OK will do later today

**Review by suriksarkisyan — APPROVED** [review]
@yzlin thanks!
I'll test and release ASAP

**Review by suriksarkisyan — APPROVED** [review]

**Pull request merged**